Fixed documentation for `sampling_ratio` option (#873)
capitalone/DataProfiler: docs/source/profiler.rst

    * **multiprocess** - Option to enable multiprocessing. Automatically selects the optimal number of processes to utilize based on system constraints.

      * is_enabled - (Boolean) Enables or disables multiprocessing
      

>     * **sample_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.

Reviewer: ```suggestion
    * **sampling_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.
```

Reviewer: its actually `sampling_ratio` per https://github.com/capitalone/DataProfiler/pull/845/files#diff-c869332fe6f1bea096dac03badb7365173fa62459962c2e9c020cb53341b1ba6R684

>     * **sample_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.

Reviewer 2: ```suggestion
    * **sample_ratio** - A percentage ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```

Author: Could you submit the same commit suggestion again? This commit suggestion is outdated after I resolved other commit suggestions.

Reviewer: done

>     * **sampling_ratio** - The ratio from 0 to 1 of how much of the input data to sample. Default value set to 0.2.

Reviewer: ```suggestion
    * **sampling_ratio** - A percentage, as a decimal, ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.
```

>     * **sampling_ratio** - A percentage, as a decimal, ranging from 0 to 1 indicating how much input data to sample. Default value set to 0.2.

Reviewer 3: Just noticed this, correct me if I'm wrong, but i don't think the value can be 0. So as oppose to from 0 to 1, `more than 0 and <= 1`. Not sure best way to type that up though. Does any other doc have a similar to mirror?

Reviewer: good catch

      * precision - Finds the precision (significant figures) within the column

        * is_enabled - (Boolean) Enables or disables precision

>       * sample_ratio - (Float) The ratio of 0 to 1 how much data (identified as floats) to utilize as samples in determining precision

Reviewer 3: I think this is a value within `Float`. We should maintain this.

Reviewer: +1 good catch -- keep this
